css-nav-1/Overview.bs

```diff
@@ -1131,105 +1155,114 @@ run the following steps:
+            unless it too is overlapped with another higher item at the same distance, recursively.
+
 </div>
 
-<div algorithm="to calculate the distance">
-To <dfn lt="calculating the distance | to calculate the distance">calculate the <var>distance</var></dfn>, the function is defined as below:
+<div algorithm="to find the shortest the distance">
+To <dfn lt="find the shortest distance">to find the shortest distance</dfn>
+between a <var>reference</var> and a <var>candidate</var>
+in direction <var>dir</var>,
+find the points <var>P1</var> and <var>P2</var>,
+respectively within the boundary boxes of the <var>reference</var> and of the <var>candidate</var>,
+that minimize the <var>distance</var> as defined as below:
 
-<pre class="prod">
+<pre><code>
     <var>distance</var> = <var>euclidean</var> + <var>displacement</var> - <var>alignment</var> - <var>sqrt(Overlap)</var>
-</pre>
+</code></pre>
 
-The meaning of each factor in the function is as follows:
+The meaning of each term is as follows:
 
 <dl dfn-type="value" dfn-for="spatial-navigation-distance-function">
     <dt><var>euclidean</var>
-    <dd>The euclidean distance between <var>P1</var> and <var>P2</var>
+    <dd>
+        The euclidean distance between <var>P1</var> and <var>P2</var>
 
     <dt><var>displacement</var>
-    <dd>The degree of displacement in <var>dir</var> between <var>searchOrigin</var> and <var>candidate</var>
-    <dd>Defined as <pre class="prod">(absolute distance on the axis orthogonal to <var>dir</var> + <var>orthogonalBias</var>) * <var>orthogonalWeight</var></pre>
-
-        * <var>orthogonalBias</var>
-            * If the <var>dir</var> is {{SpatialNavigationDirection/left}} or {{SpatialNavigationDirection/right}}, 
-            height of <var>searchOrigin</var> / 2
-            * Else if the <var>dir</var> is {{SpatialNavigationDirection/up}} or {{SpatialNavigationDirection/down}}, 
-            width of <var>searchOrigin</var> / 2
-
-        * <var>orthogonalWeight</var>:
-            * If the <var>dir</var> is {{SpatialNavigationDirection/left}} or {{SpatialNavigationDirection/right}}, 30
-            * Else if the <var>dir</var> is {{SpatialNavigationDirection/up}} or {{SpatialNavigationDirection/down}}, 2
+    <dd>
+        The degree of displacement in <var>dir</var> between the <var>reference</var> and the <var>candidate</var>,
+        defined as
+        <pre><code>
+            <var>displacement</var> = (absolute distance on the axis orthogonal to <var>dir</var> between <var>P1</var> and <var>P2</var> +
+                            <var>orthogonalBias</var>) *
+                           <var>orthogonalWeight</var>
+        </code></pre>
+
+    <dt><var>orthogonalBias</var>:
+    <dd>
+        * If the <var>dir</var> is {{SpatialNavigationDirection/left}} or {{SpatialNavigationDirection/right}},
+            the height of the axis-aligned bounding box of <var>reference</var> / 2
```

Comment:
I'm just curious - I don't understand clearly about "axis-aligned bounding box of reference".
Is it different from "bounding box of reference"?

Reply:
I mean if the box is rotated, we're not talking about the rotated bounding box, but the one that is aligned with the horizontal and vertical axises.

Reply:
Oh, now I understand that. Thanks!